src/guiClasses/MainPanelGUI.java

```diff
@@ -237,20 +191,21 @@ public static JPanel createUniPanel(String name, String info) {
 
     public static void insertionSort() {
```

Comment:
Remove static please... This should be an operation that is done on the object (i.e. UniObj.insertionSort() instead of MainPanelGUI.insertionSort(). Be creative yes, but don't throw away everything you learned in the process